test-app/app/templates/tabs/tabs-with-disable-state.hbs

```diff
@@ -0,0 +1,22 @@
+<div class='w-full max-w-md px-2 py-16 mx-auto'>
+    <TabsGroup class="flex flex-col space-x-1 rounded-xl p-1" @onChange = {{this.onChange}} as |tabs|>
```

Comment:
What are your thoughts on internalizing the onChange behavior by default?

can we abstract away the need for the consumer to configure this? (I think so!)